griptape-observability/structure_config.yaml

```diff
@@ -0,0 +1,11 @@
+version: 1.0
+runtime: python3
+runtime_version: 3.11
+build:
+  requirements_file: requirements.txt
+  cache_build_dependencies: # Configures caching (for faster deployments!)
+    enabled: false # Toggles caching
```

Comment:
would recommend enabling caching for all samples else they will fully rebuild when *any* Sample gets updated.